youtube_dl/extractor/vtm.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    urlencode_postdata,
+    compat_urllib_parse_urlencode,
+    ExtractorError,
+)
+
+
+class VTMIE(InfoExtractor):
+    """Download full episodes that require an account from vtm.be or q2.be.
+
+    The generic extractor can be used to download clips that do no require an
+    account.
+    """
+    _VALID_URL = r'https?://(?:www\.)?(?P<site_id>vtm|q2)\.be/video[/?].+?'
+    _NETRC_MACHINE = 'vtm'
+    _APIKEY = '3_HZ0FtkMW_gOyKlqQzW5_0FHRC7Nd5XpXJZcDdXY4pk5eES2ZWmejRW5egwVm4ug-'
+    _TESTS = [
+        {
+            'url': 'http://vtm.be/video/volledige-afleveringen/id/257085630173000',
```

Comment:
Ok, I'll replace it with another video. That video will go offline after a while too as all member-only videos do, but at least you'll be able to test it.